release/_releases_json.sh

```diff
@@ -58,9 +58,13 @@ function _process_new_product {
 		lc_log INFO "The version ${_PRODUCT_VERSION} is already in releases.json."
 
 		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
-	else
-		_process_product_version "${LIFERAY_RELEASE_PRODUCT_NAME}" "${_PRODUCT_VERSION}"
 	fi
+
+	local product_group_version="$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 1,2)"
+
+	jq 'map(if .productGroupVersion == "'"${product_group_version}"'" and .product == "'"${LIFERAY_RELEASE_PRODUCT_NAME}"'" then .promoted = "false" else . end)' "${releases_json}" > tmp.json && mv tmp.json "${releases_json}"
```

Comment:
- @kiwm, please format it like https://github.com/liferay/liferay-docker/blob/fec116ed4ee56b03b8791f603d139f6018247d81/release/release_gold.sh#L122-L125
- Can we remove the `else`?
- Can we rename `tmp.json` to `temp_file.json`?